EDMarketConnector.py

```diff
@@ -209,7 +209,7 @@ class AppWindow(object):
 
     def __init__(self, master: tk.Tk):  # noqa: C901, CCR001 # TODO - can possibly factor something out
 
-        self.holdofftime = config.getint('querytime') + companion.holdoff
+        self.holdofftime = cast(int, config.get_int('querytime', 0)) + companion.holdoff
```

Comment:
Enlighten me as to why this cast() is needed, given the function signature of `config.get_int()` has `-> int` ?

Reply:
Oh whoops. I thought I got all of those. At one point it was `-> Optional[int]`. I decided to drop that because it really wasnt worth the hassle and its explained in the docstring what happens on a default

Reply:
All of these should be fixed